src/States/MapViewStateDraw.cpp

```diff
@@ -251,17 +247,15 @@ void MapViewState::drawNavInfo()
 	if (isPointInRect(MOUSE_COORDS, MOVE_DOWN_ICON))
 	{
 		r.drawSubImage(mUiIcons, MOVE_DOWN_ICON.x(), MOVE_DOWN_ICON.y(), 64, 128, 32, 32, 255, 0, 0, 255);
-	}
-	else
+	} else
```

Comment:
This new code formatting style is not used anywhere else in the project. I would recommend leaving this change out for consistency. A quick search shows that everywhere else in the code base, the preferred syntax is:
```cpp
}
else
{
```

----

I actually do prefer a more compact brace style, however this really doesn't seem like the right PR to be making this change.